system/Debug/Toolbar.php

```diff
@@ -343,12 +363,19 @@ public function prepare(RequestInterface $request = null, ResponseInterface $res
 				return;
 			}
 
+			$oldKintMode        = Kint::$mode_default;
+			Kint::$mode_default = Kint::MODE_RICH;
+			$kintScirpt         = @Kint::dump('');
+			Kint::$mode_default = $oldKintMode;
+			$kintScirpt         = substr($kintScirpt, 0, strpos($kintScirpt, '</style>') + 8 );
```

Comment:
```suggestion
			$kintScript         = substr($kintScript, 0, strpos($kintScirpt, '</style>') + 8 );
```
Typo in the variable name.